ocaml/idl/datamodel.ml

```diff
@@ -663,6 +663,8 @@ let _ =
     ~doc:"The specified VBD device is not recognized: please use a non-negative integer" ();
   error Api_errors.vm_not_resident_here [ "vm"; "host" ]
     ~doc:"The specified VM is not currently resident on the specified host." ();
+  error Api_errors.vm_not_resident [ "vm" ]
+    ~doc:"The specified VM is not currently resident on any host." ();
```

Comment:
Hmm, I was wondering about the same thing. The version you proposed sounds better to me, but after reading this English Stack Exchange post I'm unsure which one to choose: https://english.stackexchange.com/questions/5464/currently-not-or-not-currently).